README.md

```diff
@@ -56,14 +46,16 @@ bundle exec middleman server
 
 See your website on `http://localhost:4567` in your browser.
 
+For more information on previewing your documentation locally, see the [Tech Docs template documentation on previewing your documentation](https://tdt-documentation.london.cloudapps.digital/create_project/preview/#preview-your-documentation).
+
 ## Tests
 
-The repository contains automated JavaScript tests that use [Jasmine][jas] test framework.
+This repository contains automated JavaScript tests that use the [Jasmine test framework][jas].
 
 To run the tests and see the results in your browser:
 
-1. Run `bundle exec rake jasmine`
-2. Go to `http://localhost:8888` in your browser
+1. Run `bundle exec rake jasmine`.
```

Comment:
This a mix of a lead-in line with a colon (used for bulleted lists) and a numbered list.